port/main.c

```diff
@@ -233,4 +247,6 @@ int main(int argc, char **argv)
 		exit(1);
 
 	mainLoop();
+
+	return 0;
```

Comment:
actually, if `mainLoop` finishes - it's probably an initialization error, so I would opt for `return 1` :).